symfit/core/printing.py

```diff
@@ -41,6 +41,15 @@ def _print_HadamardProduct(self, printer):
                       printer.doprint(self.args[1]))
 HadamardProduct._numpycode = _print_HadamardProduct
 
+if pkg_resources.parse_version(pkg_resources.get_distribution('sympy').version) \
```

Comment:
Why is this needed? Hasn't the HadamardPower printing been added to master yet?

Reply:
I'm pretty sure that's not actually in this PR, but rather something that will be resolved once the functional changes of this PR are finished, and it gets rebased onto master.

Reply:
Yes, it is. This PR was suspended for a long time and we wanted to work with the newest version. That's why I merged master.